modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json

```diff
@@ -16,11 +16,13 @@
 	"add-activation-key": "Add Activation Key",
 	"add-another-admin": "Add Another Admin",
 	"add-at-least-one-user-s-email-to-send-an-invitation": "Add at least one user's email to send an invitation.",
+	"add-contacts-to-be-notified-in-the-even-of-a": "Add contacts to be notified in the even of a",
```

Comment:
is this supposed to say `event` instead of `even`? "be notified in the even of a" i dont think makes sense?

Reply:
Thanks Kyle, it would be better, but in order not to stop "ci:forward" I'll fix it in the next subtask!